src/i18n/es.json

```diff
@@ -0,0 +1,32 @@
+[{
+  "language": "Spanish",
+  "i18n": "es",
+  "ns": "reaction-navigation",
+  "translation": {
+    "reaction-navigation": {
+      "admin": {
+        "navigation": {
+          "cancel": "Cancelar",
+          "ok": "OK",
+          "delete": "Eliminar",
+          "navigation": "Navegación",
+          "deleteTitle": "¿Eliminar elemento de navegación?",
+          "deleteMessage": "La eliminación de elementos de navegación también los elimina del árbol de navegación."
+        }
+      },
+      "navigationItem": {
+        "displayName": "Nombre para mostrar",
+        "url": "URL",
+        "isUrlRelative": "Esta URL es relativa",
+        "shouldOpenInNewWindow": "Abrir en una pestaña nueva",
+        "classNames": "Nombres de clases CSS",
+        "isVisible": "Mostrar en el tienda",
+        "isVisibleHelpText": "Haga que este elemento de navegación esté disponible en el tienda tanto para clientes como para administradores.",
+        "isPrivate": "Acceso solo para administrador",
+        "isPrivateHelpText": "Mostrar solo a los administradores. Se utiliza para organizar cambios en la navegación sin exponerla a los clientes. \"Mostrar en la tienda\" debe revisarse.",
```

Comment:
In this case, i think something like `\"Mostrar en la tienda\" debe estar seleccionado` would be a better option.

Reply:
Yes, I'm gonna correct it, thank you